doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -50,12 +50,12 @@ following Features tabs defined here.
 | Conformance                  | X             | Kubernetes conformance test |
 | None                         | X             | Kubernetes mainstream features |
 | Feature:DaemonSetUpdateSurge |               | Daemon set should surge pods onto nodes when spec was updated and update strategy is RollingUpdate |
-| Feature:IndexedJob           |               | Should create pods for an Indexed job with completion indexes |
+| Feature:[IndexedJob](https://kubernetes.io/docs/concepts/workloads/controllers/job/)          |               | Should create pods for an Indexed job with completion indexes |
 | Feature:StatefulSet          |               | Should creating a working zookeeper cluster |
 | Feature:StatefulUpgrade      |               | Stateful upgrade should maintain a functioning cluster |
-| Feature:SuspendJob           |               | Should not create pods when created in suspend state |
+| Feature:[SuspendJob](https://kubernetes.io/docs/concepts/workloads/controllers/job/)|               | Should not create pods when created in suspend state |
 | Feature:TaintEviction        |               | All pods on the unreachable node should be marked as NotReady upon the node turn NotReady AND all pods should be evicted after eviction timeout passes |
```

Comment:
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions

Why test only for "NotReady"? Why not also "NoExecute", for example?

Reply:
Humm, good question. @collivier do you have an answer to the question?